### PR TITLE
Insure log files will also be deleted after uninstall Troly plugin

### PR DESCRIPTION
**Pull request by rcdtroly**
### Why:
It appears the plugin was installed before, then removed, and a ‘log’ folder was not properly cleanup when the plugin was “uninstalled”. 

there’s data that is left in the db and not correctly cleaned up when the plugin is uninstalled.

### What:
- Execute unlink(delete) log files in `uninstall.php`
- Also fix session handler in form controller see: Alt text

### Refs:
- https://app.asana.com/0/939825968139797/1117964655163260
- https://troly-io.slack.com/archives/C4Y4R7B1T/p1554846193081600?thread_ts=1554842069.080300&cid=C4Y4R7B1T

### CODE REVIEW

File: includes/frontend/controllers/class-wp99234-forms.php [hunk 1]
```diff
@@ -17,8 +17,9 @@ class WP99234_Forms {
 
 	function __construct() {
 
-		if (!session_id()) {
-			session_start();
+	    // Using `session_status` to prevent "Cannot send session cache limiter - headers already sent" error
+        if(session_status() != PHP_SESSION_ACTIVE){
```
Comment (atifempireone) [inline includes/frontend/controllers/class-wp99234-forms.php]: @rcdtroly  fix indentation

Comment (rcdtroly) [reply includes/frontend/controllers/class-wp99234-forms.php]: @atifempireone fixed

**Comment by rcdtroly** [comment]
@atifempireone is this good to merge?

**Comment by rcdtroly** [comment]
@atifempireone good to merge?

**Pull request merged**